src/server/actions/Volunteer.ts

```diff
@@ -41,9 +43,57 @@ export async function updateVolunteer(
       updatedVolunteer
     );
   } catch (error) {
-    //throw new CMError(CMErrorType.InternalError);
+    throw new CMError(CMErrorType.InternalError);
   }
   if (!res) {
-    // throw new CMError(CMErrorType.NoSuchKey, 'Volunteer');
+    throw new CMError(CMErrorType.NoSuchKey, 'Volunteer');
+  }
+}
+
+/**
+ * Get total hours that a volunteer has volunteered
+ * @param volunteerId // Id of the volunteer
+ * @returns // Total Hours of Volunteer
+ */
+export async function getVolunteerTotalHours(volunteerId: string) {
```

Comment:
This function looks good, but it will need to be testable and tested. This also depends on a few things that haven't been finished yet so those will need to be finished before it can be tested. 

Dependencies include: create event API, sign up for event and its API, sign in and out of an event and their APIs. 
I think that's all and some of them may already be finished.